recipes/nb_anacondacloud/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+fontconfig
```

Comment:
Phantomjs from npm still seems to only work with fontconfig and freetype
from yum... issue over on nbpresent-feedstock

On 07:46, Wed, Jun 8, 2016 Mike Sarahan <notifications@github.com> wrote:

> In recipes/nb_anacondacloud/yum_requirements.txt
> <https://github.com/conda-forge/staged-recipes/pull/723#discussion_r66247944>
> :
>
> > @@ -0,0 +1 @@
> > +fontconfig
>
> I think we have fontconfig in conda Forge. Is there a reason you need to
> yum install it?
>
> https://github.com/conda-forge/fontconfig-feedstock
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/723/files/f28d3aef9f5908a486bbff2cf911e86b8a29b428#r66247944>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACxRN_11NAtKgq_fw_BHi9ylVH4yRzLks5qJrmmgaJpZM4IvMk2>
> .
>